client/my-sites/stats/site.jsx

```diff
@@ -656,7 +658,7 @@ class StatsSite extends Component {
 							className={ halfWidthModuleClasses }
 						/>
 
-						{ ! this.isModuleHidden( 'videos' ) && (
+						{ isVideoPress && ! this.isModuleHidden( 'videos' ) && (
```

Comment:
I think we could update the module visibility state directly in the `StatsNavigation` component. This way, we wouldn’t need to repeatedly check the feature (e.g., the VideoPress feature).

For example, we could update `pageModules.videos` to false when disabling the module during the feature validation step. This could work similarly to the logic in `onToggleModule` within the `StatsNavigation` component.

I haven’t tested this yet, but it seems like a cleaner approach to centralize state management for module visibility. Thoughts?

Reply:
Should this take place in the `connect()` function of the `StatsNavigation` component?